src/momento/internal/aio/_vector_index_data_client.py

```diff
@@ -108,7 +109,7 @@ async def delete_item_batch(
 
             request = vectorindex_pb._DeleteItemBatchRequest(
                 index_name=index_name,
-                ids=ids,
+                filter=F.IdInSet(ids).to_filter_expression_proto(),
```

Comment:
Previously we allowed to select items to delete with a list of ids. We now have an `id in set` filter expression which is equivalent.

Reply:
we now call fiter in proto, do we want to also change this function name?

Reply:
I don't follow? 🤔  Change the function name to what?